src/Pelagos/Bundle/AppBundle/Command/DoiComparisonCommand.php

```diff
@@ -427,4 +432,21 @@ private function sendEmail(): void
             ), 'text/html');
         $this->getContainer()->get('mailer')->send($message);
     }
+
+    /**
+     * Check if doi is orphan or duplicate.
+     *
+     * @param string  $doi     Doi identifier for the dataset.
+     * @param Dataset $dataset A dataset instance.
+     *
+     * @return boolean
+     */
+    private function isOrphan(string $doi, Dataset $dataset): bool
+    {
+        if (strtolower($dataset->getDoi()->getDoi()) !== strtolower($doi)) {
```

Comment:
gotta check if getDoi returned an instance of DOI. Getting a `Call to a member function getDoi() on null`

Reply:
Made changes 👍